2019Labs/CustomSecurityContent/documentation/lab2_openscap.adoc

```diff
@@ -23,16 +23,17 @@
 
 
 == Introduction
+Security Content Automation Protocol (SCAP) is collection of standards aiming to enable automated vulnerability and configuration compliance.
+OpenSCAP is a family of open source SCAP tools, and SCAP Security Guide (further as SSG) is collection of xml-based SCAP benchmarks and content in other formats to help with compliance configuration assessment of Red Hat Enterprise Linux machines.
```

Comment:
In the CustomSecurityContent lab we work with the upstream project ComplianceAsCode instead.

Reply:
Removed all changes to this file as per agreement.